.template/addons/crud/app/views/layouts/application.html.slim.tt

```diff
@@ -0,0 +1,25 @@
+doctype html
+html lang=I18n.locale
+  head
+    meta charset="utf-8"
+    meta name="viewport" content="width=device-width, initial-scale=1.0"
+    title= content_for?(:title) ? yield(:title) : '<%= app_name %>'
+    = csrf_meta_tags
+    = csp_meta_tag
+    = stylesheet_link_tag 'application', :media => 'all'
+    = javascript_include_tag 'application'
+
+  body class=class_names(controller_name.dasherize, action_name.dasherize)
```

Comment:
I'm unsure if we should need to keep these classes. 🤔 
```suggestion
  body
```

Reply:
These classes are actually quite usefull when you need a screen CSS class 💡 

It's added with this in mind: https://nimblehq.co/compass/development/code-conventions/css/#screens